### PR TITLE
Makes freon gas sellable

### PR DESCRIPTION
**Pull request by Toxici11i**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
fix: (TECHNICALLY AN ADDITION):Cargo had freon buying enabled, but hadnt tweaked exports to enable selling of freon
/:cl:

## About The Pull Request
Allows freon to be sold in cargo

## Why It's Good For The Game
Being unable to refund the 9000 tanks the retarded scientist bought to make a winter wonderland sucks dick, this makes it suck less dick.

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
>>>*50

yeah no, I see what this PR actually is you slime

**Comment by mkikt4743** [comment]
make it less money retard

**Comment by TrickyDickRichard** [comment]
lmao 50 points a mole did you really think this was gonna get through

**Comment by Toxici11i** [comment]
> > > > *50
> 
> yeah no, I see what this PR actually is you slime

damnit the goys saw through my plan

**Comment by KingTerrytheTerrible** [comment]
aww

**Comment by diegoxt45** [comment]
every single pr you make is just for you to abuse in game

**Comment by ghost** [comment]
diego is right

File: code/modules/cargo/exports/large_objects.dm [hunk 1]
```diff
@@ -138,4 +138,5 @@
 	worth += gases[/datum/gas/miasma][MOLES]*10
 	worth += gases[/datum/gas/tritium][MOLES]*5
 	worth += gases[/datum/gas/pluoxium][MOLES]*5
+	worth += gases[/datum/gas/freon][MOLES]*6
```
Comment (ghost) [inline code/modules/cargo/exports/large_objects.dm:141]: why should this be worth more than tritium or pluoxium?

Comment (Toxici11i) [reply code/modules/cargo/exports/large_objects.dm:141]: trit and pluxonium are piss easy to make, moreso than freon

https://github.com/HippieStation/HippieStation/pull/12496 - fixes my freon formation shitcode that made it form in ways it wasnt originally intended to

**Comment by AnthonyJNgo** [comment]
We welcome contributions but if you keep trying to slip things into the codebase that are just backdoors for yourself I'm going to remove you from the github.

**Comment by Toxici11i** [comment]
> We welcome contributions but if you keep trying to slip things into the codebase that are just backdoors for yourself I'm going to remove you from the github.

it was reduced and stop closing shit thats WIP/fixed

**Pull request closed without merging**